scalalib/worker/src/ZincWorkerImpl.scala

```diff
@@ -323,7 +363,22 @@ class ZincWorkerImpl(compilerBridge: Either[
     }
     val newReporter = reporter match {
       case None => new ManagedLoggedReporter(10, logger)
-      case r: Option[ManagedLoggedReporter] => r.get
+      case Some(r) => new ManagedLoggedReporter(10, logger) {
```

Comment:
@adadima What is the logger instance usually passed for ? I have seen you have used some other instance previously, is this configuration OK ?

Reply:
Take a look at `getCompilationLogger` in MillBuildServer.scala.
The logger would be a `ManagedLogger` that is connected to the output stream of the logger in mill's evaluator.